CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
-# Version 3.0.2-BETA
+# Version 3.0.2 GA
 * Fix App Services logback parser exceptions.
+* Log instead of throwing exception on unbridged API [#1442](https://github.com/microsoft/ApplicationInsights-Java/issues/1442)
+* Support request getId
+* Support operation getId
```

Comment:
```suggestion
* Log instead of throwing exception on unbridged API [#1442](https://github.com/microsoft/ApplicationInsights-Java/issues/1442)
* Fix role name config [#1450](https://github.com/microsoft/ApplicationInsights-Java/issues/1450)
* Support more interop with 2.x SDK ThreadContext (getId and getParentId)
```